### PR TITLE
Remove un-needed information around pgbouncer HNG-607

### PR DESCRIPTION
**Pull request by dbgreen**
## What Changed?

- Removed un-needed information about how to manually set up authentication

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing Content
- [X] This PR removes existing Content
- [ ] This PR is for a release; please add this tag:

### CODE REVIEW

**Comment by pallavisontakke** [comment]
hi @dbgreen , I've addde JIRA ticket no. to PR title, so it's linked to the ticket properly. Please see if you wish to move it elsewhere (lower) in your final commit.

File: product_docs/docs/harp/2/07_harp_proxy.mdx [hunk 1]
```diff
@@ -104,59 +104,6 @@ Do *not* use the `pgbouncer` user, as this this is used by HARP
...
-```
-
-You can also define these fields with `harpctl set proxy`:
-
-```bash
-harpctl set proxy global auth_user=pgb_auth
-```
-
 !!! Note
     This means the `postgres` or `enterprisedb` OS user that launches HARP
```
Comment (PJMODOS) [inline product_docs/docs/harp/2/07_harp_proxy.mdx:108]: This note is also somewhat misleading, one can use cert or other way of auth, not necessarily .pgpass file.

Comment (pallavisontakke) [reply product_docs/docs/harp/2/07_harp_proxy.mdx]: Note removed too.

**Review by PJMODOS — APPROVED** [review]
LGTM

**Pull request merged**